### DISCUSSION TITLE
Will there ever be sector time or tyre compound data for laps?

### DISCUSSION THREAD

**Discussion by dalton-f** [jolpica/jolpica-f1#94] [Q&A] [answered]
Sorry for another question, but pretty self-explanatory. I love using this API just wishing it had more data.

**Comment by jolpica** [comment]
At the moment we primarily source data from publically available PDFs published by the FIA, which don't have this data available.
We'd love to add data such as this in the future, but we'd need a reliable public source to obtain the data.

**Comment by harningle** [comment, marked as answer]
These data are available, but not free and thus may not be covered here. For example, if you have F1TV subscription, then livetiming in formula1 app/on f1 website allows you to see when drivers stop and which tyre compound they use. Sector times for each lap are also available there. But since it requires a subscription, we are not going to use them unless there is a free and reliable alternative data source.

That being said, please check fastf1 package: https://docs.fastf1.dev/examples_gallery/plot_strategy.html.

**Comment by harningle** [comment]
I'm reopening this as we *do* have sector times for laps, and we've already implemented part of that in https://github.com/harningle/fia-doc/issues/51. But will not prioritise this for now.
